sparkmagic/sparkmagic/tests/test_livysession.py

```diff
@@ -150,22 +151,23 @@ def test_constructor_starts_heartbeat_with_existing_session(self):
         })
         session_id = 1
         session = self._create_session(session_id=session_id, sql_created=True,
-                                       should_heartbeat=True)
+                                       should_heartbeat=True, heartbeat_timeout=0)
```

Comment:
This test is there to make sure that an existing session starts heartbeat correctly if asked to. Your modification breaks that intention. Please add a new test that reflects what you are testing for here.